README.md

```diff
@@ -42,6 +42,15 @@ See documentation for using [jib-gradle-plugin](jib-gradle-plugin#quickstart).
 
 See documentation for using [Jib Core](jib-core#adding-jib-core-to-your-build)
```

Comment:
This isn't related to this PR, but as a small suggestion, I wonder if we should organize this section up here to look more like the "Goals" section (i.e. use 3 bullets instead of 3 headers). Might look more aesthetically pleasing.

Reply:
Yeah, I think that makes the doc cleaner.